src/client/src/Client.c

```diff
@@ -93,11 +95,8 @@ STATUS createKinesisVideoClient(PDeviceInfo pDeviceInfo, PClientCallbacks pClien
                          pKinesisVideoClient->clientCallbacks.customData);
     }
 
-    // Fix-up the defaults if needed
-    // IMPORTANT!!! The calloc allocator will zero the memory which will also act as a
-    // sentinel value in case of an earlier version of the structure
-    // is used and the remaining fields are not copied
-    fixupClientInfo(&pKinesisVideoClient->deviceInfo.clientInfo);
+    // Set logger log level
+    SET_LOGGER_LOG_LEVEL(pKinesisVideoClient->deviceInfo.clientInfo.loggerLogLevel);

```

Comment:
are we making the assumption that clientInfo exists? what happens if deviceInfo is older version.